### ISSUE TITLE
Sortable Isotope - Almost there

### ISSUE THREAD

**Issue by parliament718** [metafizzy/isotope#422]
I'm trying to implement a responsive tile view using isotope.js, building on top of this fiddle built by @icholy here on GitHub:

http://jsfiddle.net/ctd7N/1/

It essentially does exactly what it should and looks great doing it.

However, that example is using Masonry. I'm trying to recreate the same fiddle as icholy using isotope (except I replaced his use of jquery-event-drag plugin with jQuery.draggable(). That switch is NOT what is causing the problem as I have tried using event-drag plugin and the same thing happens):

http://jsfiddle.net/ctd7N/59/

It's almost working. Basically the problem is that "ghost" doesn't get placed correctly into the empty space like it does with Masonry. And because the tile itself slides to the exact position of the ghost when you let go of the mouse, it ends up in the wrong place also.

This may be a little hard to see with the middle tiles, but you will see if you start from the TOP-LEFT most tile, it's almost normal. But if you drag the tiles that are more towards the right, the left offset slides them out of view. And if you drag the tiles that are more towrds the bottom, the top offset slides them out of view.

NOTE: Now here's something I found. Set top: 0 !important; left: 0 !important on the .drag-ghost class and you will see that the ghost now lines up correctly, but letting go of the mouse still causes the tile to slide out of view.

If you do this to one of the tiles that doesn't slide completely out of view and then inspect the css of that tile after it's dropped, you will see it has something set for top: __px; and left: __px.

Setting both of those properties to 0 like we did for the ghost will cause the tile to line up correctly again... At this point I realized something:

If the browser supports transitions, isotope.js will use them and only fallback to javascript top/left animations for non supporting browsers. In my browser, which DOES support transitions, all the tiles ALWAYS have top:0; left:0; and that NEVER changes even when the tiles shuffle they shuffle using a CSS transform so there is no reason for either the ghost OR tile to EVER have anything other than 0 for top/left in a supporting browser such as mine (UNLESS they are being dragged)... and yet both of them are set somehow.

To summarize: The tiles slide out of view because something sets top/left on them when it shouldn't.

I've been playing with this code for a while and can't seem to get this to work like it does in masonry. This long sought after feature would make isotope that much more awesome, and it almost working... Please help
